src/renderers/webgl/WebGLLights.js

```diff
@@ -64,7 +64,8 @@ function WebGLLights() {
 						shadow: false,
 						shadowBias: 0,
 						shadowRadius: 1,
-						shadowMapSize: new Vector2()
+						shadowMapSize: new Vector2(),
+						shadowScale: 1000,
```

Comment:
I'd call it shadowCutoffDistance, or shadowCameraFar, or something.  I think shadowScale is sort of ambiguous in terms of what it could mean -- e.g. does it mean scaling the intensity?